src/org/opendatakit/briefcase/ui/ExportPanel.java

```diff
@@ -333,6 +341,12 @@ public void enableExportButton(boolean state) {
         btnExport.setEnabled(false);
         return;
       }
+
+      if ( checkDates.isSelected() && (pickStartDate.getDate() == null || pickEndDate.getDate() == null)) {
```

Comment:
Would this be something covered in https://github.com/opendatakit/briefcase/issues/94 all at once? I saw this convention all throughout this file, was just "when in Rome"-ing.